carbon/app.py

```diff
@@ -34,20 +49,14 @@ def people():
     sql = select([persons.c.MIT_ID, persons.c.KRB_NAME_UPPERCASE,
                   persons.c.FIRST_NAME, persons.c.MIDDLE_NAME,
                   persons.c.LAST_NAME, persons.c.EMAIL_ADDRESS,
-                  persons.c.ORIGINAL_HIRE_DATE, persons.c.HR_ORG_UNIT_TITLE,
+                  persons.c.ORIGINAL_HIRE_DATE, dlcs.c.HR_ORG_LEVEL4_NAME,
                   persons.c.PERSONNEL_SUBAREA_CODE, orcids.c.ORCID]) \
         .select_from(persons.outerjoin(orcids)) \
-        .where(persons.c.EMAIL_ADDRESS != None) \
+        .where(dlcs.c.HR_ORG_UNIT_ID == persons.c.HR_ORG_UNIT_ID) \
```

Comment:
I feel like this should probably use the sqlalchemy join syntax, but I can't get it to work right at the moment. I'm going to be pretty busy today so can't really dig into it.